src/cli/java/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -149,7 +149,7 @@ public void advanceSessionWithEndpoint(String endpointId, String[] endpointArgs)
             if (s instanceof SyncScreen) {
                 try {
                     s.init(mSession);
-                    s.handleInputAndUpdateSession(mSession, "", false);
+                    s.handleInputAndUpdateSession(mSession, "", false, null);

```

Comment:
I am not very faimilar with CLI at this point, so I have parked a ticket instead - https://dimagi-dev.atlassian.net/browse/USH-1838 to get back to.